packages/rich-text/src/Toolbar/index.tsx

```diff
@@ -136,6 +138,9 @@ const Toolbar = ({ isDisabled }: ToolbarProps) => {
       <div className={styles.formattingOptionsWrapper}>
         <ToolbarHeadingButton isDisabled={isDisabled || !canInsertBlocks} />
 
+        <span className={styles.divider} />
+        <ButtonUndo />
```

Comment:
Any reasoning behind the placement? for markdown they are appended to the end for example 

<img width="725" alt="Screenshot 2024-02-22 at 10 45 53" src="https://github.com/contentful/field-editors/assets/12673605/55e33375-fc2c-494c-9bf4-194e2c80fab0">

Reply:
I think it was like that in the ticket 🙈
But I will ask

Reply:
From Fidan: I don't mind having it at the front, undo/redo are usually on top in the software (design and text editing) I used to work on